pkg/controller_workload/workload/workload_event.go

```diff
@@ -514,3 +521,25 @@ func handleAddressTypeResponse(rsp *service_discovery_v3.DeltaDiscoveryResponse)
 	}
 	return err
 }
+
+func (svc *ServiceEvent) handleAuthorizationTypeResponse(rsp *service_discovery_v3.DeltaDiscoveryResponse) error {
+	auth := &security.Authorization{}
+
+	// update resource
+	for _, resource := range rsp.GetResources() {
+		if err := anypb.UnmarshalTo(resource.Resource, auth, proto.UnmarshalOptions{}); err != nil {
+			log.Errorf("unmarshal failed, err: %v", err)
+			continue
+		}
+		log.Debugf("handle auth xds, resource.name %s, auth %s", resource.GetName(), auth.String())
+		_ = svc.rbac.UpdatePolicy(auth)
```

Comment:
If this part of the processing reports an error, it should return the error handling, right?

Reply:
Updatepolicy will not return any error, so we ignore the return value here. It seems to be confusing so I'll change the method to no return value